pytorch_lightning/loggers/trains.py

```diff
@@ -83,9 +84,13 @@ def __init__(
             auto_resource_monitoring: bool = True
     ) -> None:
         super().__init__()
-        if self._bypass:
+        if self.bypass_mode():
             self._trains = None
+            print('TRAINS Task: running in bypass mode')
```

Comment:
use rather a logger `from pytorch_lightning import _logger as log`

Reply:
@Borda what do you think? If I'll use logging it will break the doctest again, unless we change the documentation (which I'm okay with) ...